src/controller/eme-controller.js

```diff
@@ -292,6 +295,23 @@ class EMEController extends EventHandler {
     });
   }
 
+  _onKeySessionKeyStatusesChange (keySession) {
+    logger.log('Got EME keystatuseschange event, detecting license status');
+
+    // `keyStatuses` is `Map`-like object connecting to the length of `KeyIds`,
+    // but every `keyStatuses` needs to be `usable` to continue.
+    keySession.keyStatuses.forEach((status) => {
+      if (status !== 'usable') {
```

Comment:
there might be some exceptions due to legacy Browser behavior according to Google Shaka

https://github.com/google/shaka-player/blob/master/lib/media/drm_engine.js#L1366

or, should we only detect 'output-restricted' and emit a fatal error once it happens?
https://github.com/google/shaka-player/issues/652#issuecomment-272266695

Reply:
hmm, I haven't followed up what Shaka did with key status checking yet, it looks likes pretty complicated.

my initial instinct from W3C DOCUMENT is that we can rely on `usable`, because it means good to go, but I guess key status is not stable across platforms as what you mentioned in Shaka Player.

but as you suggested, we can change the condition for a lower chance to report a fatal error from HLS.

Reply:
done @springuper